src/containers/App/App.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import './App.css'
 import { Button, ButtonType, ButtonSize } from 'components'
 import { Checkbox, CheckboxType } from 'components/Checkbox'
+import { FormGroup } from 'components/FormGroup'
```

Comment:
When you add a component, you should add `export * from './FormGroup'` inside of components/index.ts just like you did with FormGroup.ts. This allows you to just `import {} from 'components'`
Now, on smaller projects, this is just an additional/optional step you can take, however on larger projects it simplifies the imports allowing you to do this:
```js
import {
  Button,
  Checkbox,
  Radiobutton,
  ...
} from 'components'
```
vs
```js
import { Button } from 'components/Button'
import { Checkbox } from 'componentsCheckbox'
import { RadioButton } from 'components/RadioButton'
```
IMHO, the prior is cleaner. Thats it :laughing:, thats my whole reasoning.

While we're on the subject. if you use export default instead of just export, you can import without {}.
`import RadioButton from 'components/RadioButton'` (assuming you did export default RadioButton)
(the method we're using is called named exports)
```suggestion
import { FormGroup } from 'components'
```

Reply:
> IMHO, the prior is cleaner. Thats it 😆, thats my whole reasoning.

I like that solution too. Import multiple things in a single statement.